apps/marketing/src/app/page.tsx

```
@@ -59,6 +59,34 @@ function Hero() {
           Open source • Self-host in minutes • Free tier
         </p>
 
+        <div className="mt-12 text-center text-xs text-muted-foreground flex flex-col items-center justify-center gap-2">
+          <p className="text-xs">Proudly sponsored by</p>
+          <a
+            href="https://coderabbit.ai/?utm_source=useSend.com"
+            target="_blank"
+          >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)